### PR TITLE
Update to version 1.15

### PR DESCRIPTION
**Pull request by bweather**
Issue: https://devtopia.esri.com/runtime/ping/issues/738

https://runtime-rtc.esri.com/view/vTest/job/vtest/job/3rdparty_libraries-interface/982/downstreambuildview/
https://runtime-rtc.esri.com/view/vTest/job/vtest/job/3rdparty_libraries-interface/983/downstreambuildview/
https://runtime-rtc.esri.com/view/vTest/job/vtest/job/3rdparty_libraries-interface/984/downstreambuildview/
https://runtime-rtc.esri.com/view/vTest/job/vtest/job/3rdparty_libraries-interface/985/downstreambuildview/

### CODE REVIEW

File: basis_universal.lua [hunk 1]
```diff
@@ -19,8 +19,15 @@ files {
...
   "*.cpp",
 
+  "encoder/**.h",
+  "encoder/**.cpp",
+  "encoder/**.c",
+
   "transcoder/**.h",
   "transcoder/**.cpp",
+
+  "zstd/**.h",
```
Comment (chri7325) [inline basis_universal.lua:29]: Looks like these are optional and may bite us if a security vulnerability is detected. We may also need to get legal approval if we want to use these. Instead, you can turn off zstd support with BASISD_SUPPORT_KTX2_ZSTD=0 as referenced https://github.com/Esri/basis_universal/commit/a8a1f7382457c9b76db824553839cf8127e225f3

**Review by chri7325 — CHANGES_REQUESTED** [review]

**Comment by chri7325** [comment]
I'm a little confused as to why our commits are showing up here even though they're already in the runtimecore branch. Was this a rebase? When moving 3rdparty up, it is best to just do a regular merge.

File: basis_universal.lua (hunk 1 above)
Comment (bweather) [reply basis_universal.lua:29]: These are needed in order to be able to rebase our SLL runtimecore branch.

**Comment by bweather** [comment]
Closing as the runtimecore branch was rebased instead of merged.
New PR: https://github.com/Esri/basis_universal/pull/8

**Pull request closed without merging**